.github/CODEOWNERS

```diff
@@ -1549,6 +1559,7 @@ x-pack/test/api_integration/apis/management/index_management/inference_endpoints
 /x-pack/test/functional_search/ @elastic/search-kibana
 
 # Management Experience - Deployment Management
+/test/plugin_functional/plugins/management_test_plugin @elastic/kibana-management
```

Comment:
It looks like this ownership is already defined in this file (in line 600) - why do we add it again?

Reply:
[Dropped!](https://github.com/elastic/kibana/pull/197980/commits/3aae4b7c8b4332c17a297f0872b0f2d749727569)